x-pack/plugins/aiops/public/components/change_point_detection/fields_config.tsx

```diff
@@ -197,15 +436,32 @@ const FieldPanel: FC<FieldPanelProps> = ({
         </EuiFlexItem>
 
         <EuiFlexItem grow={false}>
-          <EuiButtonIcon
-            disabled={removeDisabled}
-            aria-label={i18n.translate('xpack.aiops.changePointDetection.removeConfigLabel', {
-              defaultMessage: 'Remove configuration',
-            })}
-            iconType="trash"
-            color="danger"
-            onClick={onRemove}
-          />
+          <EuiFlexGroup alignItems={'center'} justifyContent={'spaceBetween'} gutterSize={'s'}>
+            <EuiFlexItem grow={false}>
+              <EuiPopover
+                id={''}
+                button={
+                  <EuiButtonIcon
+                    aria-label={i18n.translate(
+                      'xpack.aiops.changePointDetection.configActionsLabel',
+                      {
+                        defaultMessage: 'Context menu',
+                      }
+                    )}
+                    iconType="boxesVertical"
```

Comment:
We use `boxesHorizontal` in the Anomaly Explorer. Can we use that same icon here?

Reply:
Updated in https://github.com/elastic/kibana/pull/162796/commits/1161e73085abfc069b96cd2b98652c885f6162c9